wagtail/admin/views/pages/bulk_actions/delete.py

```diff
@@ -0,0 +1,58 @@
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext
+
+from wagtail.admin.views.pages.bulk_actions.page_bulk_action import PageBulkAction
+from wagtail.core import hooks
+
+
+class DeleteBulkAction(PageBulkAction):
```

Comment:
suggestion: update `action_priority` so the actions follow the listing actions (move, copy, delete, publish, unpublish)

Also it would be great if the delete bulk action button had the `.no` or `.serious` class. That is not happening on the page listing as delete is under "more"